tests/Microsoft.Identity.Web.Test/Microsoft.Identity.Web.Test.csproj

```diff
@@ -1,4 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
```

Comment:
Aren't the constant obtained from Microsoft.Identity.Web ? (transitivity?)

Reply:
Tried adding constants only to main .Web project and it doesn't seem to propagate to the dependent projects.